public/app/features/dashboard-scene/panel-edit/PanelEditor.tsx

```diff
@@ -143,6 +144,8 @@ export class PanelEditor extends SceneObjectBase<PanelEditorState> {
       width,
       height,
     });
+
+    this.publishEvent(new PersistedStateChangedEvent(), true);
```

Comment:
Since it's important to set second "bubble" param to true, an alternative is to move this to a shared util function like:

* dashboardSceneGraph.publishPersistedStateChangeEvent(this); 
* dashboardSceneGraph.notifyPersistedStateChange(this);